radis/test/benchmark/vaex_vs_pandas_performance.py

```diff
@@ -71,30 +64,52 @@ def compare_vaex_pandas_time():
 # Compare the memory performance of Pandas and Vaex
 def compare_pandas_vs_vaex_memory():
     """
-    It is for comparing the memory performance of pandas and vaex.
+    Compare memory usage of `engine="vaex"` and `engine="pandas"` in calc_spectrum.
+    Expected behavior is "vaex" using much less memory. This function takes tens of seconds to run.
+    Returns
+    -------
+    None.
     """
 
     import tracemalloc
 
-    # Change range of wav to obtain result for different number of lines , repeat same procedure both vaex and pandas
-    # Loop is not used because it gives better results for memory performance for vaex or pandas which one is used second time
-    tracemalloc.start()
-    s = calc_spectrum(
-        1000,
-        2800,  # cm-1
-        molecule="H2O",
-        isotope="1,2,3",
-        pressure=1.01325,  # bar
-        Tgas=1000,  # K
-        mole_fraction=0.1,
-        wstep="auto",
-        path_length=1,  # cm
-        databank="hitemp",  # or 'hitemp', 'geisa', 'exomol'
-        engine="vaex",
-    )
+    for engine in ["pandas", "vaex"]:
+        config["DATAFRAME_ENGINE"] = engine
```

Comment:
just as good practice, reset the final state of config["DATAFRAME_ENGINE"] at the end of the test